crates/diesel_models/src/organization.rs

```diff
@@ -58,6 +60,7 @@ impl Organization {
             modified_at,
             id: _,
             organization_name: _,
+            version: _,
```

Comment:
Either let's remove the `version` field from the `OrganizationNew` type, or let's read this value in this constructor.